impl_test.go

```diff
@@ -541,9 +542,18 @@ func TestStubGeneration(t *testing.T) {
 			iface: "github.com/josharian/impl/testdata.Interface3",
 			want:  testdata.Interface3Output,
 		},
+		{
+			iface: "Interface1",
+			want:  testdata.Interface1Output,
+			dir:   "testdata",
+		},
 	}
 	for _, tt := range cases {
-		fns, err := funcs(tt.iface, ".")
+		if tt.dir == "" {
```

Comment:
There are only three cases above. I'd prefer to just add `dir: "."` to each of them.